charts/redis/templates/redis-dc.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.managed_redis.disabled }}
```

Comment:
Can we rename the file to `DeploymentConfig.yaml` ?

Reply:
Yup, It's updated